db/seeds.rb

```diff
@@ -5,3 +5,23 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+first_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+second_user = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
+
+
+
+first_post = Post.create(author_id:first_user.id, title:"Hello", text:"This is my first post")
+second_post = Post.create(author_id:first_user.id, title:"Hello1", text:"This is my second post")
+third_post = Post.create(author_id:first_user.id, title:"Hello2", text:"This is my third post")
+fourth_post = Post.create(author_id:first_user.id, title:"Hello3", text:"This is my fourth post")
+first_post = Post.create(author_id:second_user.id, title:"Hello1", text:"This is my first post")
+second_post = Post.create(author_id:second_user.id, title:"Hello2", text:"This is my second post")
+
+
+
+Comment.create(post_id: first_post.id, author_id: first_user.id, text: "Hi Oshie!" )
```

Comment:
- [ ] Same thing with the Comment, and Like models. Let's make sure to add the `, class_name: 'User'` option on the `belongs_to :author`. Example:

```ruby
# app/models/comment.rb
belongs_to :author, class_name: 'User'
belongs_to :post
```

- The same on the Like models:

```ruby
 # app/models/like.rb
  belongs_to :author, class_name: 'User'
  belongs_to :post

```